openstf/tasks/utils/taskcontext.py

```diff
@@ -122,6 +122,17 @@ def _send_teams_message(self, exc_type, exc_info, stack_info):
                     ]
                 }
             )
+            # Add details of exceptions and pids
+            string_pids_exceptions = "\n".join(
+                [f"{key}:{value}\n" for key, value in metrics["exceptions"].items()]
+            )
+            msg["sections"].append(
+                {
+                    "facts": [
+                        ("Exceptions: pid(s)", string_pids_exceptions),
```

Comment:
Perhaps easier to read to add 1-2 newlines before the <pid>:<exception string> messages.  Not sure how it looks like this, perhaps making separate facts makes more sense or an unordered list.

Reply:
If all works well, adding it as a separate 'facts' portion, already results in some nice line breaks;
![image](https://user-images.githubusercontent.com/18208480/129913023-8d2aee89-65b4-4d2c-910c-b936ea5f184e.png)